src/restguru/restClient.ts

```diff
@@ -0,0 +1,30 @@
+import { theInternet } from "./theInternet";
+
+export interface RestClient {
+    request(requestDescription:RequestDescription):Promise<HttpResponse>
+}
+
+export interface HttpResponse {
```

Comment:
are we leaving this blank!

Reply:
i think so, we only check that there is a response in our tests for now